### DISCUSSION TITLE
IPv6

### DISCUSSION THREAD

**Discussion by limjongyeop** [GoogleChrome/lighthouse#12977] [Q&A]
To whomever it my concern, 

I noticed Google Lighthouse has several features that the organization I am contracted to, who is exploring accessibility tools for their pipeline, could potentially use. I was hoping to get answers to a few questions in order to determine compatibility with your tool and the organization I am under. 

My main question is if Lighthouse has a statement of IPv6 compliance. 
 
I also had a few other questions. If there is anyone who is able to get in touch with me regarding my specific questions, please feel free to email me at lim_jongyeop@bah.com as well. 

Thanks!

**Comment by LGNDDOLLABOUTIQUE** [comment]
Hello,and thank you for contacting us,  tell me how and where you need help
and I will gladly help you.

Sincerely

**Reply by limjongyeop** [reply]
Hi, thank you for your reply. I am not looking for help so much as questions regarding Lighthouse. Some questions I had: 

Is there any documentation on rate of false positives?
Can the install be in-house or must it be vendor hosted?
Can you tool function on a CICD pipeline? 

Thanks!

**Comment by connorjclark** [comment]
Lighthouse uses Chrome, which uses IPv6 if the OS supports it.

**Reply by limjongyeop** [reply]
Hi Connor, thanks for your reply. I was wondering if Lighthouse has a more formal statement available for IPv6 compliance? Thanks!

**Reply by patrickhulce** [reply]
No there is not.

**Reply by connorjclark** [reply]
No, in general you won't find such bureaucratic documents on open source projects. You should use your technical expertise and knowledge of Chrome to guide you here.
